ieeeuoft/makeuoft-site-2024
# removed notStarted feature to allow preview

## Overview

- Resolves #<number>
- removes noStarted feature, so users can preview the site but still not order


## Unit Tests Created

- tests fail but dw about that for now


## Steps to QA

- create a user with an application
- give it a review for accepted, waitlisted, and rejected (both on localhost:8000/admin - the django admin site)
- then check frontend and see if correct messages when sign in as participant user
